### PR TITLE
HomematicIP Cloud add cover devices platform

### PR DESCRIPTION
**Pull request by worm-ee**
**Description:**
Add HomematicIP cover platform 

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#19794

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

File: source/_components/cover.homematicip_cloud.markdown [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+title: "HomematicIP Cloud Cover"
+description: "Instructions on how to integrate HomematicIP covers within Home Assistant."
+date: 2019-04-13 13:40
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: homematicip_cloud.png
+ha_category: Cover
+ha_release: 0.86
```
Comment (MartinHjelmare) [inline source/_components/cover.homematicip_cloud.markdown]: 0.87

Comment (frenck) [reply source/_components/cover.homematicip_cloud.markdown]: Done

**Review by frenck — APPROVED** [review]

File: source/_components/cover.homematicip_cloud.markdown [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: "HomematicIP Cloud Cover"
+description: "Instructions on how to integrate HomematicIP covers within Home Assistant."
+date: 2019-04-13 13:40
```
Comment (balloob) [inline source/_components/cover.homematicip_cloud.markdown:5]: date was in the future, so it didn't show up.

**Pull request merged**